src/parquet/statistics-test.cc

```diff
@@ -397,7 +397,7 @@ TEST(CorrectStatistics, Basics) {
   ASSERT_TRUE(column_chunk4->is_stats_set());
   auto column_chunk5 = ColumnChunkMetaData::Make(
       reinterpret_cast<const uint8_t*>(&col_chunk), schema.Column(4), &version);
-  ASSERT_TRUE(column_chunk5->is_stats_set());
+  ASSERT_FALSE(column_chunk5->is_stats_set());
```

Comment:
Why is this changed needed here?

Reply:
`column_chunk5` is defined for `LogicalType::INTERVAL`, which has sort order `UNKNOWN`.
I modified this PR to not set stats for all `UNKNOWN` sort orders.